src/main/java/frc/robot/commands/RunClimbWinch.java

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.Climb;
+
+public class RunClimbWinch extends Command {
+  /** Creates a new RunClimbWinch. */
+  private Climb climb;
+
+  public RunClimbWinch(Climb climb) {
+    this.climb = climb;
+    addRequirements(climb);
+    // Use addRequirements() here to declare subsystem dependencies.
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {}
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+    climb.runWinch();
```

Comment:
we think this can/should be moved up to initialize() and not need tto be repeated every 20ms